lib/zendesk_api/middleware/response/sanitise_response.rb

```diff
@@ -0,0 +1,13 @@
+module ZendeskAPI
+  module Middleware
+    module Response
+      class SanitiseResponse < Faraday::Response::Middleware
+        def on_complete(env)
+          unless env[:body].strip.empty?
+            env[:body] = env[:body].encode('UTF-8', 'binary', invalid: :replace, undef: :replace, replace: '')
```

Comment:
I was basically going off the .travis.yml in this repo which still goes back to 1.9.3... you think we can safely drop support for pre-2.1?